modules/era/sql/synth_recipes_era.sql

```diff
@@ -18,453 +18,3476 @@ ALTER TABLE `synth_recipes`
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Steel Visor' AND ID = 12046;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Steel Walnut Lumber' AND ID = 551;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Steel-splitter' AND ID = 4555;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Stone Bangles' AND ID = 52513;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Stoneskin Torque' AND ID = 23547;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sturdy Slacks' AND ID = 30537;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sturdy Trousers' AND ID = 40506;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sun Earring' AND ID = 23028;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sun Earring' AND ID = 23057;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sun Ring' AND ID = 23523;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Sune-Ate' AND ID = 13020;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='T.M. Hooks +1 (desynth)' AND ID = 53509;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tabin Beret' AND ID = 33557;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tabin Boots' AND ID = 43032;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tabin Bracers' AND ID = 33541;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tabin Hose' AND ID = 43528;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tabin Jupon' AND ID = 34002;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tactician Magician\'s Coat +1' AND ID = 33526;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tactician Magician\'s Cuffs +1' AND ID = 33520;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tactician Magician\'s Hat +1' AND ID = 33052;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tactician Magician\'s Hooks +1' AND ID = 53511;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tactician Magician\'s Pigaches +1' AND ID = 43527;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tactician Magician\'s Slops +1' AND ID = 33518;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Taffeta Cape' AND ID = 34039;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Taikyoku Kenpogi' AND ID = 31005;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tarasque Mitts' AND ID = 42011;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tariqah -1 (desynth)' AND ID = 44037;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tariqah' AND ID = 44040;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Tathlum' AND ID = 11033;
+UPDATE `synth_recipes` SET ContentTag='TOAU' WHERE ResultName='Temple Knight Army Sword +1' AND ID = 11510;
```

Comment:
Missing a concrete date. This is at least COP from http://web.archive.org/web/20041213225043/http://ffxi.allakhazam.com/db/item.html?fitem=8493 but possibly Zilart